demo/.env

```diff
@@ -0,0 +1,5 @@
+# EIP client Demo
+NUMBER_OF_DEMO_PATIENTS=50
+EIP_DB_NAME_DEMO=openmrs_eip_mgt
+EIP_DB_USER_DEMO=openmrs_eip_mgt
+EIP_DB_PASSWORD_DEMO=password
```

Comment:
@rbuisson how are these configs passed to the demo service requiring them (in relation to the run command `docker compose -p ozone -f ../docker-compose-common.yml -f ../docker-compose-openmrs.yml -f ../docker-compose-senaite.yml -f ../docker-compose-odoo.yml -f ../docker-compose-superset.yml -f ../demo/docker-compose.yml -f ../proxy/docker-compose.yml up -d --build`)? Is the passing relative to the docker-compose file path being run or is there a merging that's happening?

Reply:
That's actually an oversight. This file has been removed now.
That would be nice to have the environment for demo to be provided in a separate file, but that brings a series of issues that we shall deal with later. I've moved this back to the root `.env` file.